lib/verk/queue_stats_counters.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Verk.QueueStatsCounters do
```

Comment:
Don't you think the manager's tests are enough? I created the `QueueStatsCounters` much more for a cosmetic reason. Just not to have the persistence logic mixed with handling the events. However, I can't really see why should we test them separately as they are practically a unit of their own.